### PR TITLE
[DOC] CLI consumer group reset

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

Updated the _Commands for managing Kafka section_ in [Getting started with the rhoas CLI](https://github.com/redhat-developer/app-services-guides/tree/main/rhoas-cli#getting-started-with-the-rhoas-cli) with consumer group command to reset partitions.

Preview: http://file.emea.redhat.com/pmellor/docs/managed-kafka-drafts/getting-started-cli.html#_commands_for_managing_kafka

### CODE REVIEW

**Comment by PaulRMellor** [comment]
@craicoverflow Hello Enda. I've added an update to the CLI Getting Started for the consumer group reset. Can you please check? If it's okay, we can merge in when ready.

File: rhoas-cli/README.adoc [hunk 1]
```diff
@@ -569,6 +569,10 @@ view the command help (for example, `rhoas kafka list -h`).
 +
 `rhoas kafka consumergroup describe`
 
+* Reset partition offsets for a particular consumer group and its members:
++
+`rhoas kafka consumergroup reset-offset`
```
Comment (craicoverflow) [inline rhoas-cli/README.adoc:574]: @PaulRMellor LGMT! ✅

**Comment by craicoverflow** [comment]
> > @craicoverflow Hello Enda. I've added an update to the CLI Getting Started for the consumer group reset. Can you please check? If it's okay, we can merge in when ready.
> 
> Thanks @craicoverflow. When will we be ready to merge in?

The reset-offset command has not been added yet. This functionality is still not possible in the Kafka Admin API so I would say we should not merge it until this is available?

**Comment by PaulRMellor** [comment]
> > > @craicoverflow Hello Enda. I've added an update to the CLI Getting Started for the consumer group reset. Can you please check? If it's okay, we can merge in when ready.
> > 
> > 
> > Thanks @craicoverflow. When will we be ready to merge in?
> 
> The reset-offset command has not been added yet. This functionality is still not possible in the Kafka Admin API so I would say we should not merge it until this is available?

Thanks. How's it looking for the sprint?

**Comment by craicoverflow** [comment]
> > > > @craicoverflow Hello Enda. I've added an update to the CLI Getting Started for the consumer group reset. Can you please check? If it's okay, we can merge in when ready.
> > > 
> > > 
> > > Thanks @craicoverflow. When will we be ready to merge in?
> > 
> > 
> > The reset-offset command has not been added yet. This functionality is still not possible in the Kafka Admin API so I would say we should not merge it until this is available?
> 
> Thanks. How's it looking for the sprint?

I'm not sure actually - I've not been working on the Kafka Admin API so not sure about timelines for having this endpoint to be available. The team have merged the change, and it is in line for the next release, but I don't know when that is planned for.

**Pull request closed without merging**